src/Symfony/Component/Messenger/Retry/RetryStrategyInterface.php

```diff
@@ -20,10 +20,10 @@
  */
 interface RetryStrategyInterface
 {
-    public function isRetryable(Envelope $message): bool;
+    public function isRetryable(Envelope $message, ?\Throwable $throwable = null): bool;
```

Comment:
This is a BC break forbidden by our [BC promise](https://symfony.com/doc/current/contributing/code/bc.html#changing-interfaces):
![Screenshot 2020-03-24 at 11 35 55](https://user-images.githubusercontent.com/7502063/77416036-ab682180-6dc3-11ea-82b5-e883926c0955.png)

We can either add a new method (using `@method` phpdoc annotation until 6.0) or introduce a child interface redefining `isRetryable` with the new optional arg.